end-to-end-tests/cypress/e2e/OSCAL/OSCAL_Load_Tests.cy.js

```diff
@@ -1,98 +1,164 @@
 const SSP_TITLE_ORIG = "Enterprise Logging and Auditing System Security Plan";
 const COMP_DEF_TITLE_ORIG = "Test Component Definition";
-const CATALOG_TITLE = "NIST Special Publication 800-53 Revision 4: Security and Privacy Controls for Federal Information Systems and Organizations";
+const CATALOG_TITLE =
+  "NIST Special Publication 800-53 Revision 4: Security and Privacy Controls for Federal Information Systems and Organizations";
 
-describe('Loading system security plans', () => {
-  it('loads SSPs by REST Mode', () => {
-    cy.navToSspEditor(SSP_TITLE_ORIG);
-    cy.waitForLoad();
-    cy.get(`[aria-label="show code"]`).click()
-    cy.scrollTo('bottom')
-    cy.contains('This is the control implementation for the system.').should('be.visible')
-  })
-  
-  it('loads system security plans by URL', () => {
-    cy.navToSspEditor(SSP_TITLE_ORIG)
-    cy.waitForLoad();
-    cy.contains("REST Mode").click()
-    cy.contains('OSCAL System Security Plan URL').first().should('exist').next().click().clear().type(Cypress.env('base_url') + "/oscal/v1/system-security-plans/cff8385f-108e-40a5-8f7a-82f3dc0eaba8")
-    cy.contains('Reload').click()
-    cy.contains(SSP_TITLE_ORIG).should('be.visible')
-    cy.scrollTo('bottom')
-    cy.contains('This is the control implementation for the system.').should('be.visible')
-  })
-  
-  it('displays Enterprise Asset Owners Party', () => {
+describe("Loading system security plans", () => {
+  beforeEach(() => {
```

Comment:
I am not a huge fan of `beforeEach` in the tests. I recognize this is repetitive, but assuming each thing we ever test for loading happens in the SSP view is not the best I think.

Reply:
I'll go ahead and reverse this change. I noticed this was done elsewhere in the file.